pkgs/development/python-modules/pythonnet/default.nix

```diff
@@ -4,7 +4,7 @@
 , pytestCheckHook
 , pycparser
 , psutil
-, dotnet-sdk
+, dotnet-sdk-sts
```

Comment:
This should be -lts, as that's what the non-prefixed version refers to

Reply:
or better, pass `dotnetCorePackager` as an arg and use the _6_0 package in the inputs list